src/components/appNavigation/AppNavigation.js

```diff
@@ -1,16 +1,19 @@
 // @flow
-import { extend } from 'lodash'
-import React, { Component } from 'react'
-import GoodWallet from '../../lib/wallet/GoodWallet'
-import { View, Text, StyleSheet, Image } from 'react-native'
+import React from 'react'
 
 import { createSwitchNavigator } from '@react-navigation/core'
-import { Link } from '@react-navigation/web'
 
 import Rewards from './Rewards'
 import BuySell from './BuySell'
 import Dashboard from './Dashboard'
 import Donate from './Donate'
+import TabsView from './TabsView'
+
+// TODO: Should we do this diferently?
```

Comment:
maybe we can consider something in the order of the `uri` (despite I tried to implement it and didn't work). I'm prefer the idea of using sprites for this kind of icons.

https://facebook.github.io/react-native/docs/images.html#source-as-an-object